src/h-basic.h

```diff
@@ -35,7 +35,7 @@
  */
 # if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) \
   || (defined(_MSC_VER) && _MSC_VER >= 1600L)
-#  define HAVE_STDINT_H
+#  define HAVE_STDINT_H 1
```

Comment:
I get a warning from SDL about redefinition of HAVE_STDINT_H. It seems like SDL and most other references define it as 1, while this definition exists without a value. Setting this to 1 gets rid of the warning, and I can't imagine it would cause problems since most other locations just check defined(...), but it might be worth testing.

Reply:
Currently, the only place HAVE_STDINT is used has a check for whether it's set and not a check on the value.  Regardless, a surer way to avoid the redefinition error would be to guard the definition in h-basic.h,

```
#ifndef HAVE_STDINT_H
#define HAVE_STDINT_H
#endif
```

Reply:
Unfortunately this specific warning is coming from SDL because they also `#define HAVE_STDINT_H 1` but the symbol has already been defined by h-basic.h. So you're right, but I'd have to patch SDL 😄

Reply:
Ah, then you could switch the include order (SDL headers first then Angband's) and add the guard to h-basic.h.  Though, your original change is simpler.

Reply:
Yes, the simple change looks good to me.